### ISSUE TITLE
[Documentation] It is too hard to figure out how to do trivial common-case sending email with twisted.mail

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4042]
|<img alt="arkanes's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| arkanes reported|
|-|-|
|Trac ID|trac#4042|
|Type|task|
|Created|2009-09-25 17:47:49Z|
|Branch|https://github.com/twisted/twisted/tree/simple-smtp-4042|

t.mail.smtp.sendmail needs to be documented a little more clearly.
Specifically, the msg parameter should clarify that email.Message refers to the stdlib package (with a link to the stdlib docs), and it should more simply say that it can be an email.Message object or the raw RF2822 message.

The current t.mail HOWTOs only include a fairly compliated SMTP client. There should be a short example answering the use case "I just want to send an email", rather than "I want to write an email client" or "I want to write an email server".

<details><summary>Searchable metadata</summary>

```
trac-id__4042 4042
type__task task
reporter__arkanes arkanes
priority__normal normal
milestone__ 
branch__branches_simple_smtp_4042 branches/simple-smtp-4042
branch_author__Orestis_Markou__thijs Orestis Markou, thijs
status__closed closed
resolution__fixed fixed
component__mail mail
keywords__documentation_twisted_mail documentation twisted mail
time__1253900869000000 1253900869000000
changetime__1295228406000000 1295228406000000
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

[#4043](https://github.com/twisted/twisted/issues/4043) was a duplicate of this, and includes a link to http://orestis.gr/blog/2009/09/26/how-send-email-twisted/ which gives some simple `sendmail` examples and documentation.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [30493]) Branching to 'simple-smtp-4042'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Added the example from the blogpost in r30499. Decided not to use his documentation because this is already covered in the `smtpclient` tutorial afaik.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

  1. Needs a news fragment
  1. Should use Twisted naming convention (ie not `to_addrs`)
  1. Need to work out copyright issue - the author should contribute it to us under the MIT license.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Replying to [exarkun](#note_5):
>   1. Need to work out copyright issue - the author should contribute it to us under the MIT license.

See the author's consent in [this comment](http://orestis.gr/blog/2009/09/26/how-send-email-twisted/#c383), a reply to a message from glyph.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra commented|
|-|-|

(In [30515]) apply coding standard, add news file. refs [#4042](https://github.com/twisted/twisted/issues/4042)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Okay, looks great.  Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_10"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [30519]) Merge simple-smtp-4042: Added a simple SMTP example, showing how to use sendmail.

Author: Orestis Markou, thijs
Reviewer: exarkun
Fixes: [#4042](https://github.com/twisted/twisted/issues/4042)
